src/api/config.rs

```diff
@@ -64,6 +64,9 @@ pub struct EncoderConfig {
   /// Flag to force all frames to be error resilient.
   pub error_resilient: bool,
 
+  /// Interval between switch frames (0 to disable)
+  pub switch_frame_interval: u64,
+
```

Comment:
We might want to include this in the C-API. See `option_match` in `src/capi.rs`.
Validating the "multiple of `group_input_len`" constraint may not be straightforward.

Reply:
It's a pretty terrible constraint in general and I want to come up with a better one.

Reply:
I guess adding a placement override could wait for that :)

Reply:
I do want to add a placement override to the API, but again it has to change the pyramid length.

Reply:
Shall we land it as-is or wait for that?

Reply:
I made it reject non-low-latency case, so this is worked around for now.